File: Fondamenti_programmazione_II/Python/Esami/Gestione_progetti_ricerca/Innovazione.py
from Progetto import Progetto

class Innovazione(Progetto):
    def __init__(self, codice, titolo, coordinatore, organizzazione, importo, aziende):
        super().__init__(codice, titolo, coordinatore, organizzazione, importo, "innovazione")
        self._aziende = aziende

    def toString(self):
        return super().__str__() +"-\t-\t"+ str(self._aziende) +"\t"+ str(self._importo*1000)

File: Fondamenti_programmazione_II/Java/EserciziEsame/GestioneProgettiRicercaProf/Innovazione.java
package EserciziEsame.GestioneProgettiRicercaProf;

public class Innovazione extends Progetto{
    private int aziende;

    public Innovazione(int codice, String titolo, String coordinatore, String organizzazione, int aziende, double importo){
        super(codice,titolo,coordinatore,organizzazione,importo);
        this.aziende=aziende;
    }

    public String toString(){
        return "innovazione\t"+super.toString()+"-\t-\t"+aziende+"1t"+importo*1000;
    }
}
